sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1278,41 +1530,78 @@ def hpat_pandas_series_len_impl(self):
 @sdc_overload_method(SeriesType, 'astype')
 def hpat_pandas_series_astype(self, dtype, copy=True, errors='raise'):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.astype
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/series_astype.py
+       :language: python
+       :lines: 27-
```

Comment:
You added Expected result before the function, may be it is better to shift to 41- line and not to show the expected result because it will be in the output?

Reply:
This is for developers who look at the code and not at the documentation.
But in the future, maybe in all functions, these comments will be removed

Reply:
I guess that this code `:lines: 27-` may show "Expected result". Code for docs starts on 41 line

Reply:
It sounds like a good idea, but I think it’s worth doing in another PR, for all such examples.